parquet/src/test/java/org/apache/iceberg/parquet/TestParquetDataWriter.java

```diff
@@ -76,7 +78,7 @@ public void createRecords() {
 
   @Test
   public void testDataWriter() throws IOException {
-    OutputFile file = Files.localOutput(temp.newFile());
+    OutputFile file = Files.localOutput(createTempFile(temp));
```

Comment:
so here we are actually creating a temp file, but in other places in this PR we are not. I think we should be consistent

Reply:
Let me make this consistent to use createTempFile(temp)